docs/runs.md

```diff
@@ -192,7 +192,11 @@ Supporting timeouts is optional but recommended.
    (i.e. `Run.Spec.Status == RunCancelled`) and or `Run.HasTimedOut()` and take
    any corresponding actions (i.e. a clean up e.g., cancel a cloud build, stop
    the waiting timer, tear down the approval listener).
-4. Once resources or timers are cleaned up it is good practice to set a
+4. A Custom Task author can differentiate between a run cancelled by an user from a
+   run cancel intiated by a pipeline run cancel by looking at the Run.Spec.StatusMessage.
```

Comment:
A few formatting/spelling notes:
* I'd go with "... a `Run` ..." rather than "... a run ..."
* "Run.Spec.StatusMessage" should be in backticks.
* "intiated" should be "initiated". =) And I'd phrase that as "a `Run` cancellation initiated by..."

Reply:
Done.
Also made some more changes to the doc on similar lines.